oak_functions/lookup/src/lib.rs

```diff
@@ -27,8 +27,7 @@ use oak_logger::OakLogger;
 // TODO(#2593): Use no_std-compatible map.
 use std::collections::HashMap;
 
-// TODO(#2579): Use no_std-compatible RwLock implementation once available.
```

Comment:
You can add `Fixes #2579` to the PR description, which would automatcially close the associated issue when the PR is merged.

Reply:
I didn't notice that the issue number was mentioned in the title, so it seems it will close the issue when merging.